features/cellular/framework/AT/AT_CellularNetwork.cpp

```diff
@@ -665,7 +665,7 @@ bool AT_CellularNetwork::has_registration(RegistrationType reg_type)
     return true;
 }
 
-nsapi_error_t AT_CellularNetwork::set_attach(int timeout)
+nsapi_error_t AT_CellularNetwork::set_attach(int /*timeout*/)
```

Comment:
I guess that "timeout" is and will not be used, so could add deprecated notice?

Reply:
good point, added deprecation